building/tooling/best-practices.md

```diff
@@ -46,19 +46,19 @@ See the [network docs](/docs/building/tooling/docker#network) for more informati
 
 ### Prefer build-time commands over run-time commands
 
-Tooling runs as one-off, short-lived Docker container:
+Tooling runs as a one-off, short-lived Docker container:
```

Comment:
If you want to go full-on proper grammar for the whole doc...

```suggestion
The track tooling runs a one-off, short-lived Docker container which executes the following steps.
```

Reply:
@IsaacG Also, I'm not sure about this one. Is the removal of 'as' intentional?

Reply:
Yes. The tooling, a collection of code, runs code and executes steps.